src/main/java/de/retest/recheck/review/ignore/MaxPixelDiffShouldIgnore.java

```diff
@@ -0,0 +1,99 @@
+package de.retest.recheck.review.ignore;
+
+import java.awt.Rectangle;
+import java.io.Serializable;
+import java.util.regex.MatchResult;
+import java.util.regex.Pattern;
+
+import de.retest.recheck.ignore.ShouldIgnore;
+import de.retest.recheck.review.ignore.io.RegexLoader;
+import de.retest.recheck.ui.descriptors.Element;
+import de.retest.recheck.ui.diff.AttributeDifference;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class MaxPixelDiffShouldIgnore implements ShouldIgnore {
```

Comment:
This is really a web-feature, isn't it? I would move this to recheck-web. However we then need to have a extensible way of adding Loaders.

Reply:
I'm not sure. While we currently only face issues with pixel diffs in recheck-web, I can't tell if such a feature wouldn't also be interesting for Swing, Android, or something else.

As you already mentioned, we ain't got no extensible way of adding loaders. Therefore, I would keep it in recheck.